### PR TITLE
additional clarity for creating a proposal

### LINKED ISSUE #605: DAO proposal create guide ETH for value is confusing
The use of ETH in the DAO proposal guide, in the create section, does not make sense as most proposals are in USDC.

> Enter an ETH Value.
> 
Better describe that the ETH goes into the Value (WEI) field and has nothing to do with the PARAMETERS field that is for the amount paid in USDC for the encode element of its array

### LINKED ISSUE #606: USDC affix zeros for proposals
Should this say add 6 zero digits to a dollar value excluding the cents? And of course 4 zeros when cents are present represented by 2 digits.
`$299     = 299000000`
`$299.50  = 299500000`
`$299.99  = 299990000`

>  Precision
> USDC uses 6 decimal places of precision as opposed to 18 that many other ERC20 tokens use. Add 6 zeros after the amount you are asking for.

### PR DESCRIPTION
**Pull request by wkande**
Closes #606 
Close #605 

Issue 606 turns out not to be all that confusing. The create proposal section allows mentions watching the create proposal video which I very helpful on this subject.

### CODE REVIEW

File: docs/guides/dao-members/proposals.md [hunk 1]
```diff
@@ -160,31 +160,42 @@ To create a new proposal using the DAO dashboard:
...
 7. Enter an **ETH Value**.
 
    > You can use zero if the target function is not `payable`.
 
-8. Enter **Parameters** which are the arguments that will be used to satisfy the
-   signature of the target contract function.
+8. In the case of step #5 above, the address
+   `0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48` is for the USDC contract. You
+   will be calling its `transfer` function, as indicated in step #6, should the
+   proposal pass. Enter **Parameters** which are the arguments that will be used
```
Comment (dcroote) [inline docs/guides/dao-members/proposals.md:176]: I think this step number 8 should begin with "Enter **Parameters** which are the..." then your addition ("In the case of step 5...") should follow

File: docs/guides/dao-members/proposals.md [hunk 2]
```diff
@@ -160,31 +160,42 @@ To create a new proposal using the DAO dashboard:
...
-In the example above, the respective Agent (primary or secondary) would be
-calling the USDC contract (`0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48`) to
-transfer 499,999 USDC to `0xF4EB52Cf9D31a...d1663d78ddDEE9`. Note that since
-`transfer(address,uint256)` transfers funds from the sender to the specified
-address, the USDC is asked to be supplied from the Agent's balance.
+   USDC uses 6 decimal places of precision as opposed to 18 that many other
+   ERC20 tokens use. Consequently, multiply the desired $USD by 10^6 to get the
+   USDC amount to enter.
+
+   ```
```
Comment (dcroote) [inline docs/guides/dao-members/proposals.md:193]: I understand the intent, but I'd leave out this table while keeping the text above it. Different geographies treat comma and period differently in numbers and there is already the example of 499,999.

**Review by dcroote — COMMENTED** [review]
A few comments from me. Since it's dashboard related, probably good to have at least another set of eyes on it- perhaps @Siegrift or @mcoetzee ?

File: docs/guides/dao-members/proposals.md [hunk 3]
```diff
@@ -160,31 +160,42 @@ To create a new proposal using the DAO dashboard:
 
    > Defines the signature of the function to call within the target contract.
    > For the target contract USDC mentioned above use
+
+   <!-- WARNING: A space wil cause the transaction to fail in solidity. -->
```
[the remark this answers is not in the document]
Comment (wkande) [reply docs/guides/dao-members/proposals.md]: I leave this comment to remind myself that the mistake described once upset a DAO member. It tells me why the space cannot be there.

File: docs/guides/dao-members/proposals.md (hunk 1 above)
Comment (wkande) [reply docs/guides/dao-members/proposals.md]: That does make sense, changed

File: docs/guides/dao-members/proposals.md (hunk 2 above)
Comment (wkande) [reply docs/guides/dao-members/proposals.md]: I am a big fan of examples but do see this one as overkill.

**Review by mcoetzee — APPROVED** [review]
It makes sense to me 🙂

**Pull request merged**